src/test/java/com/google/swarm/tokenization/orc/ORCColumnNameDoFnTest.java

```diff
@@ -25,7 +25,7 @@ public class ORCColumnNameDoFnTest {
     @ClassRule
     public static TemporaryFolder tmpFolder = new TemporaryFolder();
 
-    protected static final String PROJECT_ID = System.getenv("PROJECT_ID");
+    protected static final String PROJECT_ID = "test-project";
```

Comment:
If confirmed the ORC unit tests are working, please add in PR description as well.

Reply:
done.